pages/who_we_serve.js

```diff
@@ -11,6 +11,13 @@ import CareerServicesIcon from 'static/images/icons/Custom/career_services.svg';
 import MentorshipIcon from 'static/images/icons/Custom/mentorship.svg';
 import ScholarshipsIcon from 'static/images/icons/Custom/scholarships.svg';
 import { s3 } from 'common/constants/urls';
+import axios from 'axios';
```

Comment:
Weird that the `import/order` lint rule isn't working here 🤔I guess it doesn't take into the account the file system, and just guesses from the string itself.

Can you put the imports from axios and prop-types at the top?